src/mavsdk/plugins/camera/camera_impl.cpp

```diff
@@ -1557,17 +1560,34 @@ bool CameraImpl::should_fetch_camera_definition(const std::string& uri) const
            !_has_camera_definition_timed_out;
 }
 
+bool CameraImpl::starts_with(const std::string& str, const std::string& prefix)
+{
+    return str.compare(0, prefix.size(), prefix) == 0;
```

Comment:
Could [`string::starts_with`](https://en.cppreference.com/w/cpp/string/basic_string/starts_with) be used instead?

Reply:
It's only in C++20 but we're only on C++17 so far, unfortunately.